htdocs/themes/math4/math4.scss

```
@@ -380,6 +380,11 @@ div.showMeAnotherBox {
 h1.page-title {
 	font-size: 31.5px;
 	line-height: 35px;
+	display: inline-block;
+}
+h1.page-title + a {
+	font-size: 31.5px;
+	margin-left: 0.5rem;
 }
```

I think the help `h.page-title + a` also needs the `line-height`.  Otherwise the page title gets uneven spacing above and below it.

Also, take advantage of scss here.  So make this:
```suggestion
h1.page-title {
	font-size: 31.5px;
	line-height: 35px;
	display: inline-block;

	& + a {
		font-size: 31.5px;
		line-height: 35px;
		margin-left: 0.5rem;
	}
}
```